src/streaming/utils/Capabilities.js

```diff
@@ -291,9 +291,19 @@ function Capabilities() {
                 if (keySystemMetadata.ks.systemString) {
                     curr.keySystemConfiguration.keySystem = keySystemMetadata.ks.systemString;
                 }
-                if (keySystemMetadata.ks.systemString === ProtectionConstants.WIDEVINE_KEYSTEM_STRING) {
-                    curr.keySystemConfiguration[type] = { robustness: ProtectionConstants.ROBUSTNESS_STRINGS.WIDEVINE.SW_SECURE_CRYPTO };
 
+                let robustnessLevel = ''
+                if (keySystemMetadata.ks.systemString === ProtectionConstants.WIDEVINE_KEYSTEM_STRING) {
+                    robustnessLevel = ProtectionConstants.ROBUSTNESS_STRINGS.WIDEVINE.SW_SECURE_CRYPTO;
+                }
+                const protData = keySystemMetadata.protData
+                const audioRobustness = (protData.audioRobustness && protData.audioRobustness.length > 0) ? protData.audioRobustness : robustnessLevel;
```

Comment:
If `protData` is undefined we run into an issue here. Please check if `protData` is defined to avoid errors like `Cannot read properties of null (reading 'audioRobustness')`